R/generate_data.R

```diff
@@ -297,8 +343,9 @@ DGP <- function(
   maf_range = c(0.005, 0.010),
   method = "none",
   n = 100,
-  p_dmv = 0.33,
-  p_ptv = 0.33,
+  p_dmv = 0.40,
+  p_ptv = 0.10,
```

Comment:
What was the rationale for updating these two defaults?

Reply:
Thanks for the question, these are based on the frequencies in UK Biobank. Updated the documentation to note this.